test/integration/nft.test.ts

```diff
@@ -240,6 +240,16 @@ describe('E2E integration tests', () => {
     expect(response.looksRare).toBeDefined();
   });
 
+  it('computeRarity()', async () => {
+    const contractAddress = '0x0510745d2ca36729bed35c818527c4485912d99e';
+    const tokenId = '403';
+
+    const response = await alchemy.nft.computeRarity(contractAddress, tokenId);
+
+    expect(response).toBeDefined();
+    expect(response.length).toBeGreaterThan(0);
```

Comment:
Please update the test to check that each field in the response type is present in the first element of the array.

Reply:
Thank you for the suggestion!